kouncil-frontend/apps/kouncil/src/styles/_material.scss

```diff
@@ -443,13 +444,41 @@ mat-option > span {
 }
 
 .status-stable {
-  color: pallete.$green;
+  color: pallete.$green-50;
 }
 
 .status-dead {
-  color: pallete.$red;
+  color: pallete.$red-50;
 }
 
 .status-empty {
   color: pallete.$main-40;
 }
+
+mat-radio-button {
```

Comment:
I think that you should style material in different way, for example put class with styles inside global .scss file and then use className as attribute, then it would be easier to migrate to newer version of material in the future